contracts/libs/Utils.sol

```diff
@@ -24,3 +24,30 @@ contract AddressSet {
 		return _addressMap[_address];
 	}
 }
+
+contract AddressUintMap {
+	address[] private _addresses;
+	mapping(address => bool) private _addressMap;
+	mapping(address => uint256) private _addressValueMap;
+
+	function add(address _address, uint256 value) public {
+		_addressValueMap[_address] += value;
+		if (_addressMap[_address]) {
+			return;
+		}
+		_addressMap[_address] = true;
+		_addresses.push(_address);
+	}
+
+	function get(address _address) public view returns (uint256) {
+		return _addressValueMap[_address];
+	}
+
+	function getSumAllValue() public view returns (uint256) {
+		uint256 sum;
+		for (uint256 i = 0; i < _addresses.length; i++) {
+			sum += _addressValueMap[_addresses[i]];
+		}
+		return sum;
```

Comment:
This code is going to be computationally expensive. There is a risk because so maybe not being able to pay for gas.

Reply:
It's a may fool way, but what about creating a value that gets added each time who runs `add` method and subtracted every time who runs ` remove`(yet not there is) method?

Reply:
I think so too.
Since there is no remove method yet, only the current addition is performed.

@aggre